Makefile

```diff
@@ -22,26 +28,20 @@ all: common otelcontribcol
 e2e-test: otelcontribcol
 	$(MAKE) -C testbed runtests
 
-.PHONY: precommit
-precommit:
-	$(MAKE) gotidy
-	$(MAKE) ci
```

Comment:
I couldn't find a target called `ci` so I removed this. However, it would be nice to have a pre-commit hook, maybe using [pre-commit](https://pre-commit.com/).